src/getOperands.js

```diff
@@ -26,7 +26,8 @@
  * @return {string} - operands
  */
 function getOperands(str) {
-  // write code here
+  const result = str.match(/(-)*\d(.\d)*/g);
```

Comment:
`(-)*` - cannot be more than one `-`, use `?` instead of `*`
(.\d)* - why do you need `\d`, `.` includes it, also cannot be zero length number use `+`